CMakeLists.txt

```diff
@@ -72,10 +77,10 @@ if(CATKIN_ENABLE_TESTING)
                 src/obstacle_points.cpp
                 test/test_obstacle_points.cpp)
         target_link_libraries(obstacle_points_test ${catkin_LIBRARIES})
-	add_rostest_gtest(goal_queueing_test test/goal_queueing.test 
+	add_rostest_gtest(goal_queueing_test test/goal_queueing.test
```

Comment:
You are already doing tests for goal queueing? Awesome! Would be nice to know how to run them :))

Reply:
@JanezCim that's the unit test you also ran on your computer. This was done 3 months back.